cli/cmd/stat.go

```diff
@@ -61,6 +61,7 @@ func newCmdStat() *cobra.Command {
   * deploy/my-deploy
   * deploy/ po/
   * ds/my-daemonset
+  * job/my-job
```

Comment:
a few lines below this, you can change:
```go
  Valid resource types include:
  * daemonsets
  * deployments
  * namespaces
  * pods
  * replicationcontrollers
  * statefulsets
  * authorities (not supported in --from)
  * jobs (only supported as a --from or --to)
```
...to:
```go
  Valid resource types include:
  * daemonsets
  * deployments
  * jobs
  * namespaces
  * pods
  * replicationcontrollers
  * statefulsets
  * authorities (not supported in --from)
```

...also you can do the same thing in `tap.go` and `top.go`